spec/system/provider_interface/make_offer_spec.rb

```diff
@@ -39,7 +39,10 @@
     then_the_conditions_page_is_loaded
     and_the_default_conditions_are_checked
 
-    when_i_add_further_conditions
+    when_i_add_invalid_further_conditions
```

Comment:
Maybe this should be at the offer wizard spec level, as usually we test for the happy path 🤔